odoo/cli/load_db.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+from pathlib import Path
+from sys import argv
+
+from . import Command
+from .server import report_configuration
+from ..service.db import restore_db
+from ..tools import config
+
+
+class LoadDb(Command):
+    name = 'load_db'
+
+    def run(self, cmdargs):
+        """Loads an Odoo dump into the current system.
+
+        Aside from ``--database``, the options match (and are directly forwarded
+        too) their regular Odoo behaviour.
+        """
+        parser = argparse.ArgumentParser(
+            prog="%s %s" % (Path(argv[0]).name, self.name),
+            description=self.__doc__
+        )
+        parser.add_argument(
+            '-d', '--database',
+            help="Database to restore to, if not provided uses the dump file's"
+                 " filename (without extension"
+        )
+        parser.add_argument('-c', '--config')
+        parser.add_argument('--addons-path')
+        parser.add_argument('-r', '--db_user')
+        parser.add_argument('-w', '--db_password')
+        parser.add_argument('--pg_path')
+        parser.add_argument('--db_host')
+        parser.add_argument('--db_port')
+        parser.add_argument('--db_sslmode')
+        parser.add_argument('dump_file')
+        args = parser.parse_args(cmdargs)
+
+        # standard options parser doesn't like positional arguments, so strip
+        # the one
+        cmdargs.remove(args.dump_file)
```

Comment:
I don't like this `cmdargs.remove`

    odoo-bin load_db -d foo -c config.cfg foo

becomes

   odoo-bin load_db -d -c config.cfg foo